docs/operations/security-overview.md

```diff
@@ -264,3 +321,29 @@ As an alternative to using the basic metadata authenticator, as shown in the pre
 
 
 Congratulations, you have configured permissions for user-assigned roles in Druid!
+
+
+## Druid security trust model
+Like all other security systems, trust is the foundation of the Druid security model. Druid administrators and read-only users are trusted users. Therefore, they are not expected to act maliciously.
+
+
+Based on this expectation, Druid operates according to the following principles:
+
+From the inner most layer:
+1. Druid processes run within the system user context. They have access to the local files granted to the specified system user.
+2. The Druid ingestion system can create new processes to execute tasks. Those tasks inherit the user of their parent process. This means that any user authorized to submit an ingestion task can use the ingestion task permissions to read or write any local files or external resources that the Druid process has access to.
+
+> Note: Only grant the permission to submit ingestion tasks to trusted users because they can read and write to local file system.
```

Comment:
It's more than just the filesystem, they can make network calls as well.

```suggestion
> Note: Only grant the permission to submit ingestion tasks to trusted users because they can act as the Druid process.
```